content/patterns/checkbox/checkbox-pattern.html

```diff
@@ -14,61 +14,62 @@
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
-        <p>WAI-ARIA supports two types of <a href="#checkbox" class="role-reference">checkbox</a> widgets: dual-state checkboxes toggle between two choices -- checked and not checked -- and tri-state checkboxes, which  allow an additional third state known as partially checked.</p>
+        <h2>このパターンについて</h2>
+        <p>WAI-ARIA は、2種類の <a href="#checkbox" class="role-reference">チェックボックス</a> ウィジェットをサポートしています：二つの状態のチェックボックスは、--チェック済みとチェック済みでない--の2つの選択肢を切り替えるもので、三つの状態のチェックボックスは、これに加えて「部分的にチェックされた」という第三の状態を許容するものです。</p>
         <p>
-          One common use of a tri-state checkbox can be found in software installers where a single tri-state checkbox is used to represent and control the state of an entire group of install options.
-          And, each option in the group can be individually turned on or off with a dual state checkbox.
+          三つの状態のチェックボックスの一般的な使用方法の1つは、ソフトウェアのインストーラです。三つの状態のチェックボックスがインストールオプションのグループ全体の状態を表し、制御するために使用されます。
+          そして、グループ内の各オプションは、二つの状態のチェックボックスで個別にオン又はオフにすることができます。
         </p>
         <ul>
-          <li>If all options in the group are checked, the overall state is represented by the tri-state checkbox displaying as checked.</li>
-          <li>If some of the options in the group are checked, the overall state is represented with the tri-state checkbox displaying as partially checked.</li>
-          <li>If none of the options in the group are checked, the overall state of the group is represented with the tri-state checkbox displaying as not checked.</li>
+          <li>グループ内のすべてのオプションがチェックされている場合、全体の状態は、三つの状態のチェックボックスがチェックされることで表現されます。</li>
+          <li>グループ内の一部のオプションがチェックされている場合、全体の状態は、三つの状態のチェックボックスの一部がチェックされることで表現されます。</li>
+          <li>グループ内のオプションが1つもチェックされていない場合、全体の状態は、三つの状態のチェックボックスがチェックされていないことで表現されます。</li>
         </ul>
-        <p>The user can use the tri-state checkbox to change all options in the group with a single action:</p>
+        <p>ユーザーは、三つの状態のチェックボックスを使用して、グループ内のすべてのオプションを1つのアクションで変更することができます：</p>
         <ul>
-          <li>Checking the overall checkbox checks all options in the group.</li>
-          <li>Unchecking the overall checkbox will uncheck all options in the group.</li>
+          <li>全体のチェックボックスをチェックすると、グループ内のすべてのオプションがチェックされます。</li>
+          <li>全体のチェックボックスのチェックを外すと、グループ内のすべてのオプションのチェックが外れます。</li>
           <li>
-            And, In some implementations, the system may remember which options were checked the last time the overall status was partially checked.
-            If this feature is provided, activating the overall checkbox a third time recreates that partially checked state where only some options in the group are checked.
+            そして、一部の実装では、最後にどのオプションがチェックされていたかを記憶する場合があります。
+            この機能が提供されている場合、全体のチェックボックスを3回目に作動させると、グループ内の一部のオプションのみがチェックされている状態が再現されます。
```

Comment:
ここのthird timeは「3回目」ではなく「3回」ですね。
"that partially checked state" の訳が落ちているかも。

```suggestion
            この機能が提供されている場合、全体のチェックボックスを3回作動させると、部分的にチェックされた状態になり、グループ内の一部のオプションのみがチェックされている状態が再現されます。
```